test/SlowTests/Issues/RavenDB-16751.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FastTests.Utils;
+using SlowTests.Core.Utils.Entities;
+using SlowTests.Server.Documents.ETL;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_16751:EtlTestBase
+{
+    public RavenDB_16751(ITestOutputHelper output) : base(output)
+    {
+        
+    }
+
+    [Fact]
+    public async Task CanGetRevisionsCountInEtlScript()
+    {
+        // Arrange
+        using (var src = GetDocumentStore())
+        using (var dest = GetDocumentStore())
+        {
+            await RevisionsHelper.SetupRevisions(Server.ServerStore, src.Database);
+            await RevisionsHelper.SetupRevisions(Server.ServerStore, dest.Database);
```

Comment:
There is no need to enable revisions on the destination.

Reply:
sure, done